helper/ckan-backend-helper.php

```diff
@@ -467,6 +469,18 @@ public static function print_organisation_filter( $disable_floating = false ) {
 	}
 
 	/**
+	 * Compare organization-post_names to sort them
+	 *
+	 * @param object $a First Object to compare.
+	 * @param object $b Second Object to compare.
+	 *
+	 * @return int
+	 */
+	public static function compare( $a, $b ) {
```

Comment:
I would inline this function above (it's super small), or at least choose a better name like "organization_compare"